homeassistant/components/metoffice/sensor.py

```diff
@@ -72,7 +73,7 @@
     "wind_gust": ["Wind Gust", SPEED_MILES_PER_HOUR],
     "visibility": ["Visibility", None],
     "visibility_distance": ["Visibility Distance", LENGTH_KILOMETERS],
-    "uv": ["UV", None],
+    "uv": ["UV", UNIT_UV_INDEX],
```

Comment:
This is changing the unit of measurement, thus a breaking change?

Reply:
Yes it is, I was not sure how to come up with a suiting breaking change paragraph yet.
However, I have made one now, could you please take a look to see if it is any good?
Thanks 👍

Reply:
The description is not generic enough – this change affects more than the Météo-France integration.

Reply:
@bachya I have adjusted the breaking change paragraph to include all affected integrations. What do you think?

Reply:
Thanks for the feedback, guys!